JetAnalyzers/bin/jet_response_fitter_x.cc

```diff
@@ -138,21 +145,14 @@ int main(int argc,char**argv)
     
     if (algs.size()>0&&!contains(algs,alg)) continue;
 
```

Comment:
Why did you remove what was here? Was it due to the switch from "UPDATE" to "RECREATE"? Is it impossible to keep "UPDATE" and use your workflow? A switch is not impossible for us, but I want to make sure the change is justified.